lawnchair/src/app/lawnchair/LawnchairLauncher.kt

```diff
@@ -157,11 +159,13 @@ class LawnchairLauncher : QuickstepLauncher(), LifecycleOwner,
     val gestureController by lazy { GestureController(this) }
     private val defaultOverlay by lazy { OverlayCallbackImpl(this) }
     private val prefs by lazy { PreferenceManager.getInstance(this) }
+    private val preferenceManager by lazy { PreferenceManager.getInstance(this) }
```

Comment:
Why not use `prefs`?

Reply:
Uh, I didn't notice its already added because preferenceManager2 was added by its full name. Ill fix it now.